lib/commands/deploynodeapp.js

```diff
@@ -72,6 +80,31 @@ var descriptor = defaults.defaultDescriptor({
     name: 'Preserve policies from previous revision',
     shortOption: 'P',
     toggle: true
+  },
+  runtime: {
```

Comment:
Oh, I got you.  Since `--runtime` is a term we use in Turbo for choosing your base image, maybe we could use something different.  I remember liking `--backend=[edge-services|trireme]` but the more I think about it, we could use a flag like `--edge-services` as well.  The more I think about it, I like the flag because it's really just a simple boolean.  I don't see us having more backends for this stuff.